modules/oadp-odf-cpu-memory-requirements.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-odf-cpu-memory-requirements_{context}"]
+= Adjust Ceph CPU and memory requirements based on collected data
+
+The following recommendations are based on observations of performance made in the scale and performance lab. The changes are specifically related to {odf-first}. If working with {odf-short}, consult the appropriate tuning guides for official recommendations.
+
+[id="oadp-odf-config-cpu-memory-requirements_{context}"]
+== CPU and memory requirement for configurations
+
+Backup and restore operations require large amounts of CephFS `PersistentVolumes` (PVs), and to avoid Ceph MDS pods restarting with an `out-of-memory` (OOM) error, the following configuration is suggested:
```

Comment:
```suggestion
Backup and restore operations require large amounts of CephFS `PersistentVolumes` (PVs). To avoid Ceph MDS pods restarting with an `out-of-memory` (OOM) error, the following configuration is suggested:
```